spidermon/contrib/validation/utils.py

```diff
@@ -1,5 +1,13 @@
 from __future__ import absolute_import
 import re
+import logging
+import json
+import six
+from six.moves.urllib.parse import urlparse
+from six.moves.urllib.request import urlopen
+from scrapy.utils.misc import load_object
```

Comment:
:lipstick: Please, keep standard ordering:

```
from __future__ import absolute_import

import json
import logging
import re

import six
from scrapy.utils.misc import load_object
from six.moves.urllib.parse import urlparse
from six.moves.urllib.request import urlopen
```

Also, it may be better to use `from six import string_types` and move that import together with the other `six` imports.

Reply:
I re-ordered just for the sake of pep8 compliance, this change was made a while back, since then I have removed all linters.

Reply:
Implemented